web/themes/custom/usagov/scripts/callCenterWaitTimeLookup.js

```diff
@@ -0,0 +1,84 @@
+jQuery(document).ready(async function () {
+  "use strict";
+
+  var waittime = (function() {
+    if (jQuery("html[lang|='en']").length ||
+        jQuery("html[lang|='es']").length) {
+      var domain = window.location.origin.split('.')[0].split('//')[1];
+      var domainMap = {
+        'www': 'cms',
+        'cms': 'cms',
+        'beta': 'cms',
+        'beta-stage': 'cms-stage',
+        'cms-stage': 'cms-stage',
+        'beta-dev': 'cms-dev',
+        'cms-dev': 'cms-dev',
+      };
+      jQuery.ajax({
+        "url": "https://" + domainMap[domain] + ".usa.gov/wait-time",
```

Comment:
We don't have a case here for localhost, which is kind of sad. My dev site is trying to call https://undefined.usa.gov/wait-time 

(http://localhost/wait-time is returning the string "false", which I guess is fair since the script isn't working.)